config/install/views.view.alert_banners.yml

```diff
@@ -978,13 +944,185 @@ display:
           multi_type: separator
           separator: ', '
           field_api_classes: false
+        field_alert_end_date:
```

Comment:
Good addition of date fields to the REST export. The fields are properly configured with `html_datetime` format for JavaScript parsing.